modular_skyrat/modules/modular_weapons/code/medguns.dm

```diff
@@ -110,147 +110,147 @@
+	ammo_type = /obj/item/ammo_casing/energy/medical // This is the ammo type that all mediguns come with.
 	medicell_examine = TRUE
 
 /obj/item/weaponcell/medical/oxygen
 
-//Tier I cells//
-//Brute I//
+// Tier I cells
+// Brute I
 /obj/item/weaponcell/medical/brute
-	name = "Brute I Medicell"
-	desc = "A small cell with a red glow. Can be used on Mediguns to unlock the Brute I Functionality."
+	name = "brute I medicell"
+	desc = "A small cell with a red glow. Can be used on mediguns to unlock the brute I functionality."
 	icon_state = "Brute1"
 	ammo_type = /obj/item/ammo_casing/energy/medical/brute1/safe
 	secondary_mode = /obj/item/ammo_casing/energy/medical/brute1
 	primary_mode = /obj/item/ammo_casing/energy/medical/brute1/safe
 	toggle_modes = TRUE
 
-//Burn I//
+// Burn I
 /obj/item/weaponcell/medical/burn
-	name = "Burn I Medicell"
-	desc = "A small cell with a yellow glow. Can be used on Mediguns to unlock the Burn I Functionality."
+	name = "burn I medicell"
+	desc = "A small cell with a yellow glow. Can be used on mediguns to unlock the burn I functionality."
 	icon_state = "Burn1"
 	ammo_type = /obj/item/ammo_casing/energy/medical/burn1/safe
 	secondary_mode = /obj/item/ammo_casing/energy/medical/burn1
 	primary_mode = /obj/item/ammo_casing/energy/medical/burn1/safe
 	toggle_modes = TRUE
-//Toxin I//
+// Toxin I
 /obj/item/weaponcell/medical/toxin
-	name = "Toxin I Medicell"
-	desc = "A small cell with a green glow. Can be used on Mediguns to unlock the Toxin I Functionality."
+	name = "toxin I medicell"
+	desc = "A small cell with a green glow. Can be used on mediguns to unlock the toxin I functionality."
 	icon_state = "Toxin1"
 	ammo_type = /obj/item/ammo_casing/energy/medical/toxin1
-//End of Tier I Cells/
-//Tier II Cells/
-//Brute II//
+// End of Tier I Cells
+// Tier II Cells
+// Brute II
 /obj/item/weaponcell/medical/brute/better
-	name = "Brute II Medicell"
-	desc = "A small cell with a intense red glow. Can be used on Mediguns to unlock the Brute II Functionality."
+	name = "brute II medicell"
+	desc = "A small cell with a intense red glow. Can be used on mediguns to unlock the brute II functionality."
 	icon_state = "Brute2"
 	ammo_type = /obj/item/ammo_casing/energy/medical/brute2/safe
 	secondary_mode = /obj/item/ammo_casing/energy/medical/brute2
 	primary_mode = /obj/item/ammo_casing/energy/medical/brute2/safe
-//Burn II//
+// Burn II
 /obj/item/weaponcell/medical/burn/better
-	name = "Burn II Medicell"
-	desc = "A small cell with a intense yellow glow. Can be used on Mediguns to unlock the Burn II Functionality."
+	name = "burn II medicell"
+	desc = "A small cell with a intense yellow glow. Can be used on mediguns to unlock the burn II functionality."
 	icon_state = "Burn2"
 	ammo_type = /obj/item/ammo_casing/energy/medical/burn2/safe
 	secondary_mode = /obj/item/ammo_casing/energy/medical/burn2
 	primary_mode = /obj/item/ammo_casing/energy/medical/burn2/safe
-//Toxin II//
+// Toxin II
```

Comment:
We should just do that for the whole file, I won't make a billion comments about it, you know what I mean.
```suggestion

// Toxin II
```